lib/src/kbasesearchengine/main/IndexerWorker.java

```diff
@@ -82,13 +75,14 @@ public IndexerWorker(
             final IndexingStorage indexingStorage,
             final TypeStorage typeStorage,
             final File tempDir,
-            final LineLogger logger) {
+            final LineLogger logger) throws IOException {
```

Comment:
Very minor, but unless the method sig can fit on a single line IMO it's more readable if the exceptions get their own line. That separates them from the args.